product_configurator/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     "author": "Pledra, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "website": "https://github.com/OCA/product-configurator",
-    "depends": ["account", "stock"],
+    "depends": ["account"],
```

Comment:
Shouldn't it at least depend from `product`?

Reply:
It's included in the chain, account depends on product https://github.com/odoo/odoo/blob/13.0/addons/account/__manifest__.py#L18

Reply:
OK. 
I'm totally for lighter dependencies. 
Still, Travis build is red, so something else is probably missing.